api/src/pcapi/admin/custom_views/offerer_view.py

```diff
@@ -113,5 +116,20 @@ def update_model(self, form: Form, offerer: Offerer) -> bool:
         if result:
             for email in _get_emails_by_offerer(offerer):
                 update_external_pro(email)
+            try:
+                zendesk_sell.update_offerer(offerer)
+            except SystemExit as error:
+                logging.warning(f"Error while updating offerer : {error}", extra={"offerer": offerer.id})
```

Comment:
J'aurais tendance à préciser pour une identification plus rapide dans les logs : `Error while updating offerer in Zendesk Sell: {error}"`